android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -360,4 +363,12 @@ internal class CommonKtTests {
         verify(exactly = 1) { onResult.onError(mockErrorMap) }
     }
 
+    @Test
+    fun `getPaymentDiscount returns an error`() {
+        val error = getPaymentDiscount()
+        assertNotNull(error)
```

Comment:
I usually try to leave out null check assertions if I'm going to be asserting the value of the object later on. Is there extra value to null checking in kotlin?

Reply:
Good point, getPaymentDiscount will always return a not null object actually since it's marked as non-nullable in Kotlin. Removing!!